client/src/components/navigations/MainStackNavigator/index.tsx

```diff
@@ -96,7 +100,6 @@ const onMessageSubscription = graphql`
         }
         memberships(excludeMe: false) {
           user {
-            id
```

Comment:
I think it is better to have the `id` here for future maintenance. The uniqueness is very useful in organizing caches.

Reply:
Thank you for your advice.

I will add this opinion on the code.